bot/common/graphql.py

```diff
@@ -400,14 +403,22 @@ async def update_user(data, where):
     return result
 
 
-async def update_user_display_name(display_name, id):
-    return await update_user({"display_name": {"set": display_name}}, {"id": id})
+async def update_user_display_name(id, display_name):
+    return await update_user(
+        {"display_name": {"set": display_name}, "name": {"set": display_name}},
+        {"id": id},
+    )
 
 
-async def update_user_twitter_handle(twitter_handle, id):
+async def update_user_twitter_handle(id, twitter_handle):
     try:
         return await update_user(
-            {"twitter_user": {"create": {"username": twitter_handle}}}, {"id": id}
+            {
+                "twitter_user": {
+                    "create": {"username": twitter_handle, "name": twitter_handle}
```

Comment:
We are not capturing name for twitter, probably best to leave this empty

Reply:
backed it out